### ISSUE TITLE
How to install the media scanner for the latest releases, why is it not included? Or do I need it? 

### ISSUE THREAD

**Issue by robotswillwin** [CasparCG/server#1483]
Hello!

I haven't found anything good on this, but as I understand it, neither the new official releases nor the NRK-fork contains the media scanner anymore (why?) and to be able to run tls, cls, thumbnail retrieve and so on I need to install it separately, right? Am I missing something here?

So I've tried to download the latest media scanner (https://github.com/CasparCG/media-scanner/releases/tag/v1.1.0) and then put it (scanner.exe and prebuilds-folder) inside the same folder as the latest caspar server but i cannot get it to work. 

The only solution that works now is to start the scanner.exe that comes with the 2.3.3 LTS inside the 2.3.3 installation and run the lastest casparcg.exe from its folder, but it seems kind of ugly :D. I've tried to move that version (2.3.3 LTS) of the scanner to the latest caspar server release as well (and copy all files that I think are needed) but with no success.

So this is the output I get from the scanner.exe that is shipped with 2.3.3 LTS when moved to the C:\casparcg-server-5299709de8ac968ec7e5566b5c3147882b894ddc-windows -folder (I get the same result with the NRK-fork)

```
[2023-05-25T07:45:36.402Z] INFO (scanner/21324 on dsgncg02): request completed
req: {
"id": 214,
"method": "GET",
"url": "/db/_media/SMR_QUESTIONS%2F2%2F1_MATEMATIK_SVAR?",
"headers": {
"accept": "application/json",
"content-type": "application/json",
"host": "localhost:8000",
"connection": "close"
},
"remoteAddress": "::ffff:127.0.0.1",
"remotePort": 52878
}
res: {
"statusCode": 200,
"header": "HTTP/1.1 200 OK\r\nX-Powered-By: Express\r\nContent-Type: application/json\r\nContent-Length: 214\r\nETag: W/\"d6-rUIum+LnTGCttvNwe1qvAOhnTHk\"\r\nVary: Accept-Encoding\r\nDate: Thu, 25 May 2023 07:45:36 GMT\r\nConnection: close\r\n\r\n"
}
responseTime: 1
[2023-05-25T07:45:36.403Z] ERROR (scanner/21324 on dsgncg02):
req: {
"id": 213,
"method": "GET",
"url": "/thumbnail/SMR_QUESTIONS%2F2%2F1_matematik_SVAR",
"headers": {
"host": "localhost:8000",
"accept": "*/*",
"connection": "close"
},
"remoteAddress": "::1",
"remotePort": 52877
}
err: {
"type": "TypeError",
"message": "Cannot read property 'thumb.png' of undefined",
"stack": "TypeError: Cannot read property 'thumb.png' of undefined\n at module.exports.app.get.wrap (C:\\snapshot\\media-scanner\\src\\app.js:0:0)\n at <anonymous>\n at process._tickCallback (internal/process/next_tick.js:188:7)"
}
[2023-05-25T07:45:36.403Z] ERROR (scanner/21324 on dsgncg02): request errored
req: {
"id": 213,
"method": "GET",
"url": "/thumbnail/SMR_QUESTIONS%2F2%2F1_matematik_SVAR",
"headers": {
"host": "localhost:8000",
"accept": "*/*",
"connection": "close"
},
"remoteAddress": "::1",
"remotePort": 52877
}
res: {
"statusCode": 500,
"header": "HTTP/1.1 500 Internal Server Error\r\nX-Powered-By: Express\r\nDate: Thu, 25 May 2023 07:45:36 GMT\r\nConnection: close\r\n\r\n"
}
err: {
"type": "Error",
"message": "failed with status code500",
"stack": "Error: failed with status code500\n at ServerResponse.onResFinished (C:\\snapshot\\media-scanner\\node_modules\\pino-http\\logger.js:40:33)\n at emitNone (events.js:111:20)\n at ServerResponse.emit (events.js:208:7)\n at onFinish (_http_outgoing.js:723:10)\n at afterWrite (_stream_writable.js:454:3)\n at _combinedTickCallback (internal/process/next_tick.js:144:20)\n at process._tickCallback (internal/process/next_tick.js:180:9)"
}
responseTime: 3
```

**Comment by Julusian** [comment]
The scanner isnt included currently because I had to redo the automated building from scratch and have been putting off setting it up to also include the scanner.
But it should be as simple as download the release from that other repository, extract it and run. I cant explain the issues you are seeing currently

**Comment by Julusian** [comment]
The scanner is now included in the builds as of 39883b521df001e7752e1796b3cfa3a04085cbf6
